game/scripts/npc/abilities/centaur_double_edge.txt

```diff
@@ -39,7 +39,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "strength_damage"                                 "60 80 100 120 140 180"
+        "strength_damage"                                 "60 80 100 120 180 240"
```

Comment:
This ability on Centaur is the only good scaling ability on Centaur. This is also a percentage of his strength. Not cool. Please revert.

Reply:
ok